File: Server.NET.Core/Toto.MovieInfo/src/PersistenceModel/Movie.cs
using System;
using System.Collections.Generic;

namespace Toto.MovieInfo.PersistenceModel
{
    public class Movie
    {
        public Guid Id { get; set; }
        public Guid? GenreId { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string ReleaseDate { get; set; }
        public virtual Genre Genre { get; set; }
        public virtual ICollection<MovieActor> MovieActors { get; set; } = new List<MovieActor>();
    }
}

File: Client.Angular7/src/app/movies/movie-model/movie.ts
import { Actor } from "./actor";
import { Genre } from "./genre";
import { Poster } from "./poster";

export interface Movie {
    id: string;
    name: string;
    description: string;
    releaseDate: string;
    rating: number;
    poster: Poster;
    genre: Genre;
    actors: Actor[];
}